docs/mint.json

```diff
@@ -910,7 +910,7 @@
       "source": "/latest/community"
     },
     {
-      "destination": "/latest/develop/interact-with-data",
+      "destination": "/latest/develop",
```

Comment:
this should get covered in `develop/blocks` which I'll revisit today or tomorrow if you want to proactively link there

Reply:
Cool. Done in latest commit.

Reply:
@discdiver i don't think you pushed it yet

Reply:
My bad, should be there now.